core/src/main/java/com/airbnb/aerosolve/core/transforms/FloatFamilyToDenseTransform.java

```diff
@@ -0,0 +1,88 @@
+package com.airbnb.aerosolve.core.transforms;
+
+import com.airbnb.aerosolve.core.FeatureVector;
+import com.airbnb.aerosolve.core.util.Util;
+import com.typesafe.config.Config;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * cross one float family with the other and out a 2D dense feature.
+ * if fieldsName2 missing, this is a self cross
+ * self cross use key's alphabetical order to determine order
+ * so you can add 1_ 2_ in front of features to manipulate order.
+ */
+public class FloatFamilyToDenseTransform implements Transform {
```

Comment:
rename this? it does not reflect the fact that you are doing some cross internally